### ISSUE TITLE
Could this possibly be working now?

### ISSUE THREAD

**Issue by holodrum** [13xforever/gog-galaxy-2.0-cover-manager#1]
After one of the updates this year (not sure which one), Galaxy 2 stopped resetting my manually changed covers (which it used to do), so I wonder if a cover manager might be possible at this point?

**Comment by 13xforever** [comment]
nothing has changed for me, unfortunately

**Comment by holodrum** [comment]
That is really weird, if your Galaxy is up to date, it should function like this: https://imgur.com/a/sawvo1f

I captured the specific windows (instead of the entire screen), so for a quick overview and to avoid confusion, the left bottom window is just a folder with a couple of covers I downloaded, the bottom left is the Galaxy 2 data folder.

As shown, it permanently overwrites the picture in the Galaxy data folder when you import your own cover and also redownloads the default cover (from IGDB) and overwrites the custom one, if you remove your own.

Even after uninstalling and reinstalling G2 (as long as I don't manually delete the data folder), the custom covers are kept and automatically switched out in a reinstall.

**Edit:** Obviously, this might be completely irrelevant in regards to this application, I don't claim to understand how any of this would work or what plans you had for this! 
Just thought I would ask if anything had changed since Galaxy at least doesn't remove the manually imported covers anymore and the Galaxy users are in desperate need of some kind of cover manager/importer as doing it per game like this is extremely slow :)

**Comment by 13xforever** [comment]
idk how big your library is, but I have 7k+ games and Galaxy is really struggling with large amount of games, so it invalidates any custom changes pretty much immediately. I had some changes to stick (both metadata and covers), but there's no pattern that I could see.

There's no time stamping or checksums or any other mechanism to distinguish if some data was modified in the db, so I don't have the faintest how it is supposed to work.

I'm checking it like twice a year and I'll look at it again around January when I'll have some time available, but I wouldn't hold my breath.

**Comment by holodrum** [comment]
Yeah, they really need to work on optimizing Galaxy, I have even more games than that (getting close to 13k) and every time I launch Galaxy, it has to (for some reason) "import"/load my games, it takes a while and it makes no sense.

I guess I'm just lucky that my Galaxy (so far) is keeping all of my custom artwork since yours isn't, I'm curious as to why though, especially since my library is so big and Galaxy is struggling in other areas.

I'll just continue using Playnites’ metadata plugin to automatically switch covers for all of my games and manually add them to Galaxy (although I've been trying to figure out a way to automate this).

Anyway, I appreciate you taking the time to answer and explain here, hopefully, someday it's possible to get a cover manager working for Galaxy, as the current way to do it is absolutely horrible.

**Comment by 13xforever** [comment]
It usually keeps the covers for a while _unless_ you click on the game to see the details (activity/achievements/etc), that's when Galaxy usually refreshes all data erasing customizations for me.

**Comment by 13xforever** [comment]
Yeah, I just looked at the db, and tested the changes, and nothing changed. There's nothing in db still to indicate that user has custom artwork, it still simply saves new images in cache with random name, and that's it.

It resets any custom changes immediately, so I have no idea how they're tracking if the game has custom artwork and it wasn't modified.
